### PR TITLE
Injective test

### PR DESCRIPTION
**Pull request by Zetazzz**

### CODE REVIEW

File: .telescope.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "protoDirs": [
     "./protos"
   ],
-  "outPath": "./libs/cosmos-types",
+  "outPath": "./networks/injective/codegen-injective",
```
Comment (Zetazzz) [inline .telescope.json:5]: the .telescope.json file in the project root is generated every time telescope runs. it only summarizes the config of the last run of telescope. so I think we should delete it and put it into gitignore.

File: networks/injective/.env.development.example [hunk 2]
```diff
@@ -0,0 +1,3 @@
+
+# can not start with 0x, so remove 0x from the beginning
```
Comment (Zetazzz) [inline networks/injective/.env.development.example:2]: should this include? Has it put in gitignore?

File: networks/injective/codegen-injective/index.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export * from './injective/client'
```
Comment (Zetazzz) [inline networks/injective/codegen-injective/index.ts:1]: Does this whole codegen folder for injective have to be checked into main?

File: networks/injective/.env.development.example (hunk 2 above)
Comment (sdqede) [reply networks/injective/.env.development.example:2]: This is a .example file, deleting it is a wise choice, thanks for your advise

File: networks/injective/codegen-injective/index.ts (hunk 3 above)
Comment (sdqede) [reply networks/injective/codegen-injective/index.ts:1]: Good idea, I've deleted the folder and add it to gitignore

File: .telescope.json (hunk 1 above)
Comment (sdqede) [reply .telescope.json:5]: Good idea, deleted and added to gitignore

**Pull request merged**